src/main/java/teammates/common/datatransfer/questions/FeedbackContributionQuestionDetails.java

```diff
@@ -239,7 +239,7 @@ public String getQuestionResultStatisticsJson(
     }
 
     @Override
-    public List<String> validateResponsesDetails(List<FeedbackResponseDetails> responses, int numRecipients) {
+    public List<String> validateResponsesDetails(List<FeedbackResponseDetails> responses) {
```

Comment:
I won't advise trying to change this here. You'll soon realize how much change this would necessitate.

Reply:
Yes. This param is quite baked in to the interface and it would require all the dependencies to adapt to the new interface... If necessary, I can make a PR to fix it.